server/models/remote_provider.go

```diff
@@ -518,6 +533,9 @@ func (l *RemoteProvider) SaveK8sContext(token string, k8sContext K8sContext) (K8
 	}
 
 	resp, err := l.DoRequest(cReq, token)
+	if resp == nil {
+		return K8sContext, ErrUnreachableRemoteProvider(err)
```

Comment:
Here err can be nil. Everywhere, first do the err nil check and then resp nil check please.

Reply:
thanks ,fixed it.